util/check_translation.py

```diff
@@ -61,10 +62,10 @@ def translate_regex(regex, trans):
         for old, new in trans['~effectNames'].items():
             did_work = did_work or re.search(old, line)
             line = re.sub(old, new, line)
-    for old, new in trans['replaceText'].items():
+    for old, new in sorted(trans['replaceText'].items(), key=lambda x: -len(x[0])):
```

Comment:
oh we need longer ones first, otherwise, the shorter one will replace the leading parts of longer ones if it's a prefix of the longer.
IE, with this 
```
"Larboard Wave Cannon": "左舷齐射·波动炮",
"Larboard Wave Cannon Surge": "左舷齐射·零式波动炮",
```
the `Larboard Wave Cannon Surge` will be replaced by the shorter first, which will become `左舷齐射·波动炮 Surge` and the longer ones cannot match it.

Reply:
this prefix problem will also affect `timeline.js`, see https://github.com/quisquous/cactbot/pull/424/commits/523d8f672b1d90d07fa865153a959c330e5b64d3 for more information